client-data/board.css

```diff
@@ -1,7 +1,19 @@
+@font-face {
+	font-family: 'Roboto';
+	font-style: normal;
+	font-weight: 400;
+	src: url('fonts/roboto-v30-latin-regular.eot'); /* IE9 Compat Modes */
+	src: local(''),
+	url('fonts/roboto-v30-latin-regular.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
+	url('fonts/roboto-v30-latin-regular.woff2') format('woff2'), /* Super Modern Browsers */
+	url('fonts/roboto-v30-latin-regular.woff') format('woff'), /* Modern Browsers */
+	url('fonts/roboto-v30-latin-regular.ttf') format('truetype'), /* Safari, Android, iOS */
+	url('fonts/roboto-v30-latin-regular.svg#Roboto') format('svg'); /* Legacy iOS */
```

Comment:
I'd like to avoid embedding large blobs in this repo.
Can we use system fonts ?

Reply:
Unfortunentally Roboto is only possible as extra font.
But Arial almost looks like the same. We could also use Roboto directly from google, but this I want to avoid because then we are not independent.

But I think deliver a font is like the images we have in the repo. It`s not unusal to deliver the correct font in a software project.

Reply:
I just took a look. the size is only ~90kb

Reply:
Can we just use the system's default ? On the frontend wejust use 

```css
font-family: sans-serif;
font-weight: 300;
```

which I think looks decent and well-integrated everywhere.